tile-compiler/fractal-detailer/src/main/java/io/github/pfwikis/FastNoiseLite.java

```diff
@@ -54,68 +54,40 @@
 // /*FNLfloat*/ float
 // /*FNLfloat*/ double
 
-public class FastNoiseLite
```

Comment:
I would like to keep this file unchanged. It is actually an external dependency (that is only distributed as a java file, not a maven package). Changing it makes it harder to see if anything changed in the source.

Reply:
I saw that. It comes from here: https://github.com/Auburn/FastNoiseLite/tree/master/Java

If you want to see if something changes, can't that be done by looking at the last commit date, or the commit history in general, of that file?